ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs

```diff
@@ -221,7 +221,7 @@ private void LoadComputedCurveRows(List<UncertainPairedData> computedCurves)
             for(int i =0;i<computedCurves.Count;i++)
             {
                 UncertainPairedData upd = computedCurves[i];
-                CurveComponentVM computeComponent = new CurveComponentVM(StringConstants.STAGE_DAMAGE, StringConstants.STAGE, StringConstants.DAMAGE);              
+                CurveComponentVM computeComponent = new ComputeComponentVM(StringConstants.STAGE_DAMAGE, StringConstants.STAGE, StringConstants.DAMAGE,DistributionOptions.HISTOGRAM_ONLY);              
```

Comment:
Careful. Richard renamed ComputeComponentVM to CurveComponentVM.

Reply:
Yeah, i saw that. I had some resolves that i worked through. I must have missed that one.

Reply:
I hope that was a reasonable name refactor.

Reply:
It's a better name for sure. I never liked ComputeComponentVM. I just couldn't come up with something better. Something Something emotional invalidation, misnaming, and off by 1 errors, something something.